builtin/logical/mssql/path_config_lease.go

```diff
@@ -78,7 +87,7 @@ func (b *backend) pathConfigLeaseRead(
 	return &logical.Response{
 		Data: map[string]interface{}{
 			"ttl":     leaseConfig.TTL.String(),
-			"ttl_max": leaseConfig.TTLMax.String(),
+			"max_ttl": leaseConfig.TTLMax.String(),
```

Comment:
Yes. This can be a problem. How about returning both fields, one with a deprecation notice?